.drone.star

```diff
@@ -422,3 +443,115 @@ def sonarAnalysis(ctx, phpVersion = DEFAULT_PHP_VERSION):
         "trigger": trigger,
     }]
     return result
+
+def composerInstall():
+    return [{
+        "name": "composer-install",
+        "image": OC_CI_PHP % DEFAULT_PHP_VERSION,
+        "environment": {
+            "COMPOSER_HOME": "%s/.cache/composer" % dir["base"],
+        },
```

Comment:
I think we have to use this for all steps that do `composer install` in order for restored cache to work